### DISCUSSION TITLE
How to keep pinned tabs from hiding

### DISCUSSION THREAD

**Discussion by ronsmith** [piroor/treestyletab#3577] [Q&A]
The row of pinned tabs keeps hiding. I don't want that. How do I make it stop.

**Comment by piroor** [comment]
Did you mean that pinned tabs are unexpectedly hidden and you want they are kept visible? 
TST Options => checking "Unlock Expert Options" => "Appearance" => "Suppress visual gap of the sidebar contents produced by temporarily shown/hidden toolbars on following cases" may affect to the trouble.

And there is another solution: right-click on the bookmarks toolbar and mark it as "Always Show".
